### PR TITLE
Finishes [#187355106] Ft profile page settings

### PR DESCRIPTION
**Pull request by pacifiquemboni**
1.What does this PR do?
Profile Page Settings

2.Description
Once user user is done with registration he/she will be able to see his/her profile(includes personal information and billing address) and he/she will be able to update it by adding additional information such as gender, birthdate etc.

3.Acceptance Criteria
I want to be able to Edit my profile, but i should NOT be able to edit another users profile
relevant pivotal tracker story https://www.pivotaltracker.com/story/show/187355106

4.steps followed to test this feature
i)clone develop branch git clone https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn.git
ii)Checkout to the branch using git checkout ft-profile-page-setting
iii)setup environment variables (.env file) following .env.example file
PORT="your port number"
DATABASE_URL="your_database_url"
NODE_ENV="your_database_level"(it shoulbe development, test or production)
iv)run npm run build to convert .ts files into .js format
v) run npm migrate:all to migrate provided models in you database
vi)run npm run dev to start the server

5.use :
npm run test to run tests and npx jest --coverage to see test coverage
http://localhost:your_port_number/docs to see swagger documenta

### CODE REVIEW

**Review by lilika67 — APPROVED** [review]

**Review by Christian-Ishimwe — APPROVED** [review]

File: .vscode/settings.json
Comment (UmuhireJessie) [inline .vscode/settings.json:1]: Remove this folder

File: src/models/userModels.ts
Comment (UmuhireJessie) [inline src/models/userModels.ts:1]: Remove this file if you are not using it

File: src/tests/index.test.ts
Comment (UmuhireJessie) [inline src/tests/index.test.ts:1]: Remove this file if you are not using it

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Good job @pacifiquemboni, consider the following:
1. Work on the comments give
2. Resolve the conflicts and conversations

**Review by lilika67 — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**